### DISCUSSION TITLE
How to indicate an issue is blocked

### DISCUSSION THREAD

**Discussion by jordandsullivan** [unitaryfund/mitiq#2293] [Q&A] [answered]
Do we use any specific workflow to indicate that an issue is blocked? e.g. a tag?

**Comment by natestemen** [comment, marked as answer]
We have [`monitor-for-external-solution`](https://github.com/unitaryfund/mitiq/labels/monitor-for-external-solution) which has generally been used to indicated blocked. A list of all labels can be found [here](https://github.com/unitaryfund/mitiq/labels).

**Reply by jordandsullivan** [reply]
I'm thinking of issues like #2245 which need additional discussion, so I wasn't sure if [monitor-for-external-solution](https://github.com/unitaryfund/mitiq/labels/monitor-for-external-solution) would be appropriate. 

Actually I think the tag [needs-agreed-design](https://github.com/unitaryfund/mitiq/labels/needs%2Fagreed-design) would work in this case.

**Comment by cosenal** [comment]
I wish Github supported linking issues across projects/orgs, which [it doesn't at the moment](https://github.com/orgs/community/discussions/4928).
